packages/edit-site/src/components/page-pages/index.js

```diff
@@ -234,6 +234,7 @@ export default function PagePages() {
 						value: id,
 						label: name,
 					} ) ) || [],
+				// enableFilterHiding: false,
```

Comment:
This is for testing purposes: either remove the comment or uncomment it before merging this PR. Same below.